Sources/swift-doc/Supporting Types/Pages/FooterPage.swift

```diff
@@ -37,6 +40,8 @@ struct FooterPage: Page {
     }
 
     var html: HypertextLiteral.HTML {
+        dateFormatter.locale = datesLocale
```

Comment:
I've also noticed that there is a discrepancy with the Markdown footer, that uses `"Generated at \(timestamp) using [swift-doc](\(href)) \(SwiftDoc.configuration.version)."` while it could use `"Generated on \(dateString)"` instead.
WDYT?